examples/mugen/retrieval/train.py

```diff
@@ -0,0 +1,67 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+from examples.mugen.data.mugen_datamodules import MUGENDataModule
+from examples.mugen.data.mugen_dataset import MUGENDatasetArgs
+from examples.mugen.retrieval.model import VideoCLIPLightningModule
+from hydra.utils import instantiate
+from omegaconf import OmegaConf
+from pytorch_lightning import Trainer
+from pytorch_lightning.callbacks import ModelCheckpoint
+from torchmultimodal.transforms.bert_text_transform import BertTextTransform
+from torchmultimodal.transforms.video_transform import VideoTransform
+
+
+def get_yaml_config():
+    cli_conf = OmegaConf.from_cli()
+    if "config" not in cli_conf:
+        raise ValueError(
+            "Please pass 'config' to specify configuration yaml file for running VideoCLIP training"
+        )
+    yaml_conf = OmegaConf.load(cli_conf.config)
+    conf = instantiate(yaml_conf)
+    return conf
+
+
+def train():
+    args = get_yaml_config()
+
+    dataset_args: MUGENDatasetArgs = args.dataset_args
+    datamodule = MUGENDataModule(
+        dataset_args,
+        text_transform=BertTextTransform(
+            **vars(args.datamodule_args.bert_text_transform)
+        ),
+        video_transform=VideoTransform(**vars(args.datamodule_args.video_transform)),
+        batch_size=args.datamodule_args.batch_size,
+        num_workers=args.datamodule_args.num_workers,
+        shuffle=args.datamodule_args.shuffle,
```

Comment:
Does this evaluate to `False` as the default in the yaml? In MUGEN `shuffle` is `True` during training.

Reply:
https://github.com/mugen-org/MUGEN_baseline/blob/b45da493f3ddbcced52d44eee02d443f90842474/retrieval/train.py#L21